doc/classes/InputEventMIDI.xml

```diff
@@ -90,7 +90,7 @@
 			The pressure of the MIDI signal. This value ranges from 0 to 127. For many devices, this value is always zero.
 		</member>
 		<member name="velocity" type="int" setter="set_velocity" getter="get_velocity" default="0">
-			The velocity of the MIDI signal. This value ranges from 0 to 127. For a piano, this corresponds to how quickly the key was pressed, and is rarely above about 110 in practice.
+			The velocity of the MIDI signal. This value ranges from 0 to 127. For a piano, this corresponds to how quickly the key was pressed, and is rarely above about 110 in practice. Note that some MIDI devices may send a [code]MIDI_MESSAGE_NOTE_ON[/code] message with zero velocity and expect this to be treated the same as a [code]MIDI_MESSAGE_NOTE_OFF[/code] message, but device implementations vary so Godot reports event data exactly as received.
```

Comment:
```suggestion
			The velocity of the MIDI signal. This value ranges from 0 to 127. For a piano, this corresponds to how quickly the key was pressed, and is rarely above about 110 in practice. Note that some MIDI devices may send a [constant MIDI_MESSAGE_NOTE_ON] message with zero velocity and expect this to be treated the same as a [constant MIDI_MESSAGE_NOTE_OFF] message, but device implementations vary so Godot reports event data exactly as received.
```
Please do the change by amending the commit with `git commit --amend` (and then `git push --force`) to keep it as a single commit ready to merge.

Reply:
I made these suggested changes and force pushed. Also corrected the email address. Thank you!